clusterDeployer.py

```diff
@@ -135,6 +139,25 @@ def setup_all_vms(h: host.Host, vms: List[NodeConfig], iso_path: str) -> List[Fu
 
     return futures
 
+def deploy_microshift(cluster_name:str, nodes: List[NodeConfig] ,external_port:str) -> None:
```

Comment:
function should get a single `NodeConfig` since the function ignores the rest anyway.